main.py

```diff
@@ -123,34 +120,33 @@ def createDataDict(checkledLink):
     pdf_link = f"https://dergipark.org.tr{shortLink}"
     dataDict['Yayın PDF'] = pdf_link
     
-    # Parsing the data has been done
-    # I'm adding the data to the data_list
+    # I'm adding Parsed data to the data_list
     data_list.append(dataDict)
 
-    print(f"{counter}. Article created [{checkCounter}. Article]")
+    print(f"{counter}. Article created")
 
+    # OUTPUT TO TXT FILE
     # Writing the information into txt file (Used in articles directory)
     # But its not necessary for the main project
-
-    # with open("article{counter}.txt",'w',encoding='utf-8') as f:
-    #     f.write(f"Makale Başlığı: {dataDict['Makale Başlığı']}\n")
-    #     f.write(f"Özet: {dataDict['Özet']}\n")
-    #     f.write(f"Yazar isimleri: {dataDict['Yazar İsimleri']}\n")
-    #     f.write(f"Yayın Yılı: {dataDict['Yayın Yılı']}\n")
-    #     f.write(f"Dergi ismi: {dataDict['Dergi İsmi']}\n")
-    #     f.write(f"Yayın sayfa url: {dataDict['Yayın Sayfa URL']}\n")
-    #     f.write(f"Yayın pdf linki: {dataDict['Yayın PDF']}\n")
- 
-
-# Checking articles
-# This made for adding controls to the data extraction
-# For example:
-# If you want to extract only articles with 'fizik' in their label, you can customize this function for it
-# Just need to add 'if' statement to the 'for label in labels:' loop (E.g: if 'fizik in label':) 
-
-# Because the purpose of this project is to extract all articles, I didn't add any controls
-def checkFunc(magazineLink):
-    global checkCounter
+    # ↓ Uncomment the code below to use
+    # OutputToTXTFile(dataDict)
```

Comment:
This comment section is unnecessarily long. Need to update it to make it shorter.